examples/java/com/google/census/CensusRunner.java

```diff
@@ -36,11 +39,42 @@
 
   public static void main(String args[]) {
     System.out.println("Hello Census World");
-    System.out.println("Default Tags: " + Census.getCensusContextFactory().getDefault());
```

Comment:
Do you mean that the PR should have been broken into multiple commits to make the diffs clearer?

The main difference is that the new code uses the gRPC Context to keep track of the Census tags.  We could add back the `CensusScope` class, implemented using the gRPC context, to make this code clearer.